workers/unity/Packages/io.improbable.gdk.core/.codegen/Source/Generators/Core/UnityEcsViewManagerGenerator.cs

```diff
@@ -33,6 +33,7 @@ public static CodeWriter Generate(UnityComponentDetails componentDetails)
                         {
                             evm.Line($@"
 private WorkerSystem workerSystem;
+private ComponentSystem lookupSystem;
```

Comment:
Please use the most specific type here.
`private SpatialOSReceiveSystem spatialOSReceiveSystem;`